src/blosc2/proxy.py

```diff
@@ -82,9 +92,10 @@ def aget_chunk(self, nchunk: int) -> bytes:
 
         Notes
         -----
-        This method is optional, and only available if the source has an async `aget_chunk` method.
+        This property is optional, and only available if the source has an async `aget_chunk` method.
```

Comment:
property -> method

Reply:
Fixed in 3cf9ac234c9d1f287e4f6ff1936f747f859b6713